presto-iceberg/src/main/java/com/facebook/presto/iceberg/util/StatisticsUtil.java

```diff
@@ -89,7 +90,7 @@ public static TableStatistics mergeHiveStatistics(TableStatistics icebergStatist
             }
             statsBuilder.setColumnStatistics(columnHandle, mergedStats.build());
         });
-        return calculateAndSetTableSize(statsBuilder).build();
+        return calculateAndSetTableSize(statsBuilder).setConfidenceLevel(HIGH).build();
```

Comment:
Maybe the confidence level after merging shouldn't higher than any one of `icebergStatistics` and `hiveStatistics`, what do you think?